status-im/status-react
# chore: disable non selectable bridge assets

fixes: https://github.com/status-im/status-mobile/issues/20842

**account page bottom sheet**
<img src="https://github.com/user-attachments/assets/da064e6c-80a8-4086-b00c-5cfee486c945" width="200px" />

**home page bottom sheet**
<img src="https://github.com/user-attachments/assets/f08897fb-3f3f-42e6-8589-06c3c56e9178" width="200px" />

**bridge page**
<img src="https://github.com/user-attachments/assets/1b6429d5-7bb4-49a4-a5b8-98ac61f8a3dd" width="200px" />

To test:
check the bridge entry points,
wallet home long press on asset
account page long press on asset
bridge flow from account page cta

only a ETH and stable coins should be bridgeable.
